tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -148,9 +148,9 @@ object Discover {
     } catch {
       case t: Throwable =>
         throw new BuildException(s"""Output from '$versionCommand' unexpected.
```

Comment:
Maybe wrapping this one like the one below would look nicer.

Reply:
by "wrapping", do you really mean "unwrapping" or "unfolding"? inserting a break after the opening paren?